identity-apps-core/pom.xml

```diff
@@ -686,6 +686,20 @@
                     <autoReleaseAfterClose>true</autoReleaseAfterClose>
                 </configuration>
             </plugin>
+
+            <plugin>
```

Comment:
Check if the indentation is consistent

Reply:
Fixed with [9adbbd6](https://github.com/wso2/identity-apps/pull/6746/commits/9adbbd68db08be1a9a82e085da9fac6c58ebd36f)